docs/services/seafile.md

```diff
@@ -0,0 +1,107 @@
+# Seafile
+
+Seafile ist eine sichere, Open-Source-Plattform für die Synchronisation, Freigabe und Zusammenarbeit von Dateien, die
+sowohl eine On-Premises- als auch eine Cloud-basierte Bereitstellungsoption bietet.
+
+```yaml
+version: '3.9'
+
+services:
+  mariadb:
```

Comment:
did you check if it is supporting nowadays postgresql too?

Reply:
No. Blocks it.

Reply:
Alright, so mariadb ist still required